charts/aws-vpc-cni/values.yaml

```diff
@@ -44,6 +46,7 @@ nodeAgent:
   enableIpv6: "false"
   metricsBindAddr: "8162"
   healthProbeBindAddr: "8163"
+  resources:
```

Comment:
nit: let's use `resources: nil` and add a comment that when not specified, node agent container resources are inherited from `Values.resources`

Reply:
Sorry, I'm not sure what you mean by `resources: nil`? The idiomatic pattern for an empty value in a Helm chart is either `key:` or `key: {}`, not defining the key is bad practice as it breaks tooling. The documentation for how to use this is in the README.

Reply:
Let's use `resources: {}` to make it cleaner and more explicit